spring-cloud-stream-dapr/spring-cloud-stream-binder-dapr-grpc/src/main/java/io/dapr/spring/cloud/stream/binder/messaging/DaprHeaders.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2022 The Dapr Authors
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.dapr.spring.cloud.stream.binder.messaging;
+
+/**
+ * Dapr internal headers for spring messaging messages.
+ */
+public final class DaprHeaders {
+
+	private DaprHeaders() {
+	}
+
+	/**
+	 * The contentType tells Dapr which content type your data adheres to when
+	 * constructing a CloudEvent envelope.
+	 *
+	 * <p>
+	 * Type: String
+	 */
+	public static final String CONTENT_TYPE = "contentType";
```

Comment:
Is there a way to reuse these from the core SDK?

Reply:
Deleted relevant content and replaced these with the method in the Dapr JAVA SDK